03/app.js

```diff
@@ -0,0 +1,9 @@
+const btnsList = document.querySelectorAll('button');
+
+btnsList.forEach(btn => {
```

Comment:
Tu nie trzeba bo mamy pętlę - wiele elementów, jak będzie ich 0 to po prostu pętla się nie wykona ;)